ystia/yorc
# Feature/gh 364 ansible update

# Pull Request description

## Description of the change

### What I did

Updated ansible version to 2.7.9 to address known critical severity security vulnerability detected in ansible version 2.7.2 previously used.

### How to verify it

On a linux host,
* either install ansible 2.7.9 as root running `pip install ansible==2.7.9`
* or just activate a virtualenv as your user.

Then, use yorc bootstrap to bootstrap a setup.
Check the bootstrapped setup in using ansible 2.7.9 running `pip list`

### Description for the changelog

Upgrade Ansible to 2.7.9 ([GH-364](https://github.com/ystia/yorc/issues/364))

## Applicable Issues

https://github.com/ystia/yorc/issues/364
